resources/vars-default.scss

```diff
@@ -30,3 +30,10 @@ $color_rating_target: #700;
 $path_to_root: '.';     // relative path from style.css to STATIC_ROOT
 
 @mixin vars-img {}
+
+// custom colours
+$color_user_submission_activity0: #ddd;
```

Comment:
Is there a reason this is after the mixin, or can we keep things ordered a little bit better?

Reply:
Was thinking to separate one-off colours but I can move it with the rest.

Reply:
Done